tests/e2e/.mocharc.js

```diff
@@ -11,22 +11,22 @@ module.exports = {
   recursive: true,
   require: 'node_modules/jet/platform/node',
   spec: [
-    '../packages/app/e2e/**/*.e2e.js',
+    // '../packages/app/e2e/**/*.e2e.js',
```

Comment:
this looks great! But we need the other tests to run as well - I frequently forget and commit this file with this exact sort of temporary change also also :-)

Reply:
wow. That is an oversight -- I had problems running the functions tests (somehow they require an out-of repo file). This should never have been committed. I'll revert this.

Reply:
>  I had problems running the functions tests (somehow they require an out-of repo file)

:thinking: that's odd - I just did #5945 which switched e2e functions tests to use the emulator but I thought I'd stitched everything up nicely. Maybe I missed something. I'll peel this to a new issue for tracking, thanks for mentioning it